lib/get_parser.py

```diff
@@ -156,6 +156,17 @@ def get_base_parser(add_condor_epilog: bool = False) -> argparse.ArgumentParser:
     return parser
 
 
+def get_submit_parser(add_condor_epilog: bool = False) -> argparse.ArgumentParser:
```

Comment:
Could you please add a quick docstring here explaining that this is the common parser used by all submission binaries (condor_submit, jobsub_submit)?